.github/scripts/PrCommentCheck.main.kts

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env kscript
+
+import kotlin.system.exitProcess
+
+val prDescription = args.firstOrNull().orEmpty()
+val storyRegex = Regex("https:\\/\\/app.shortcut.com\\/simpledotorg\\/story\\/[0-9]+\\/[A-Za-z0-9_-]+")
+
+println("Checking PR description!")
+
+if (prDescription.contains(storyRegex)) {
+  println("PR description contains a valid story link.")
```

Comment:
Why do we need printing statements?

Reply:
Just for action logs, that's all.

Reply:
and we are not exposing any sensitive content, so I think this should be good for now.

Reply:
Got it. Let's delete this later then

Reply:
I don't think there is any reason to delete them. The print statements are basically just a message we show when an action is successful or a failure. Since the exit process cannot have a message, we are doing it with this.